### DISCUSSION TITLE
Supabase Realtime stress test?

### DISCUSSION THREAD

**Discussion by iaudouard** [supabase/supabase#11558] [Questions]
Hey, does anyone have an idea how scalable supabase realtime is? Can it handle thousands of concurrent connections and micro processes for example?

**Comment by w3b6x9** [comment]
@fusunnn we've run Realtime stress tests internally and we'll publish them in the near future.

> Can it handle thousands of concurrent connections

Yes, it can.

> micro processes

What do you mean?
